app/frontend/src/styles/application/hiringStaff/scrollable-panel.scss

```diff
@@ -1,64 +0,0 @@
-// The overarching goal: we want the shadow to always be present whenever the pane is scrollable.
```

Comment:
This PR appears to regress this shadow behaviour, so that it only kicks in within a certain maximum device-width.